### PR TITLE
added team communication section

### PR DESCRIPTION
**Pull request by AloisReitbauer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ This repository is associated with the [Distributed Trace Context Community Grou
 
 Specification for tracing context propagation format.
 
+## Team Communication 
+
+Overview of team related communication channels
+
+* GitHub issues for any specification related issues
+* Mailing List for general discussions. Please subscribe to [public-trace-context@w3.org](public-trace-context@w3.org)
```
Comment (yurishkuro) [inline README.md]: recommend this as the link instead, where people can actually subscribe: http://lists.w3.org/Archives/Public/public-trace-context/

**Review by yurishkuro — APPROVED** [review]

**Comment by codefromthecrypt** [comment]
Agree with yuri about link. Otherwise lgtm

**Comment by AloisReitbauer** [comment]
Can fix later. I did this as a quick fix in the editor. Don't want to check out the branch now locally. I make the adjustment once this is approved.

**Comment by codefromthecrypt** [comment]
I clicked the edit button and changed it for ya

**Review by codefromthecrypt — APPROVED** [review]

**Pull request merged**